### PR TITLE
Marc@follow up challenges

### PR DESCRIPTION
**Pull request by marcbeunardeau88**

### CODE REVIEW

**Comment by dannywillems** [comment]
Nit: also a typo in the first commit message

File: kimchi/src/circuits/berkeley_columns.rs [hunk 1]
```diff
@@ -12,15 +12,61 @@ use serde::{Deserialize, Serialize};
...
-    BerkeleyChallenges, ColumnEnvironment, Constants, Domain, FormattedOutput,
-};
+use crate::circuits::expr::{ColumnEnvironment, Constants, Domain, FormattedOutput};
 
 use crate::circuits::wires::COLUMNS;
 
 use std::collections::HashMap;
 
-use super::expr::BerkeleyChallengeTerm;
+/// The challenge terms used in Berkely
```
Comment (dannywillems) [inline kimchi/src/circuits/berkeley_columns.rs:21]: Typo

**Review by dannywillems — COMMENTED** [review]
LGTM for me so far

File: kimchi/src/circuits/berkeley_columns.rs (hunk 1 above)
Comment (marcbeunardeau88) [reply kimchi/src/circuits/berkeley_columns.rs]: fixed up

**Review by dannywillems — APPROVED** [review]

**Review by dannywillems — CHANGES_REQUESTED** [review]
The PR doesn't compile.

**Pull request merged**